libkbfs/tlf_journal.go

```diff
@@ -1442,6 +1443,9 @@ func (j *tlfJournal) doOnMDFlushAndRemoveFlushedMDEntry(ctx context.Context,
 		if err != nil {
 			return err
 		}
+		// Remember to make the info file again if more data comes
```

Comment:
Hmm...might be cleaner just to recreate the file here. My original intention here was just to take care of (i.e., remove) any orphaned files.

Reply:
This removes the whole subdir though.  If we re-created the info file here, all TLF subdirs would stick around forever.